azurerm/internal/services/dns/resource_arm_dns_ns_record.go

```diff
@@ -33,6 +31,9 @@ func resourceArmDnsNsRecord() *schema.Resource {
 			Delete: schema.DefaultTimeout(30 * time.Minute),
 		},
 
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
```

Comment:
Should we use the NsRecord parser here as with the other resources?

Reply:
Good catch - TY!